Project.toml

```diff
@@ -18,6 +18,7 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 ReverseDiff = "37e2e3b7-166d-5795-8a7a-e32c996b4267"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 TimerOutputs = "a759f4b9-e2f1-59dc-863e-4aeb61b1ea8f"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```

Comment:
we shouldn't add Documenter to the main project

Reply:
fixed